.eslintrc.js

```diff
@@ -2,10 +2,15 @@
 
 module.exports = {
   root: true,
+  parserOptions: {
+    ecmaVersion: 2021,
+    sourceType: 'script',
+  },
```

Comment:
why adding this - it should have been set by `plugin:node/recommended`. I've just put it to the last, to overwrite the config in other presets.

Reply:
Oh, I see. [unicorn/recommended](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/configs/base.js) sets `sourceType: 'module'` so I had to set it back to `sourceType: 'script'` here.

This came up because I alphabetized the list in `extends`. I actually think it's better practice for us to be explicit about our `sourceType` instead of relying on the ordering of `extends`. We shouldn't have to worry about what's happening in `extends`.

But I can change it back if you prefer.

Reply:
I do like eslint-plugin-node provides a good default for most node.js apps, but surely, it does not hurt to add our own. :)